game_engine.py

```diff
@@ -288,15 +340,41 @@ async def run_game(engine_interface):     # run a full game of diamant
 #     print(success)
 
 
+# class TestEngineInterface:
```

Comment:
Would like to see this either somewhere else as actual code or removed altogether